### PR TITLE
BAU - Update bundle

### PR DESCRIPTION
**Pull request by JHjava**
- Run bundle update

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -352,6 +357,7 @@ GEM
 
 PLATFORMS
   ruby
+  x86_64-darwin-17
```
Comment (jakubmiarka) [inline Gemfile.lock]: is this now required?

Comment (JHjava) [reply Gemfile.lock]: No idea. It was just what bundle update generated.

Comment (rhowe-gds) [reply Gemfile.lock]: Looks like it's pulled in a MacOS version of `libv8`

File: Gemfile.lock [hunk 2]
```diff
@@ -193,12 +198,11 @@ GEM
...
       stud
-    prometheus-client (0.10.0.pre.alpha.1)
-      concurrent-ruby
-    psych (3.1.0)
-    public_suffix (3.0.3)
-    puma (3.12.1)
-    rack (2.0.6)
+    prometheus-client (0.10.0.pre.alpha.2)
+    public_suffix (3.1.1)
+    puma (4.1.0)
```
Comment (rhowe-gds) [inline Gemfile.lock]: Interesting read: https://schneems.com/2019/06/26/puma-4-new-io-4-your-server/

File: Gemfile.lock [hunk 3]
```diff
@@ -102,24 +105,25 @@ GEM
...
-    geckodriver-helper (0.23.0)
+    geckodriver-helper (0.24.0)
       archive-zip (~> 0.7)
     globalid (0.4.2)
       activesupport (>= 4.2.0)
-    govuk-lint (3.11.0)
-      rubocop (~> 0.64)
+    govuk-lint (3.11.5)
+      rubocop (~> 0.72)
+      rubocop-rails (~> 2)
```
Comment (rhowe-gds) [inline Gemfile.lock]: have fun

File: Gemfile.lock (hunk 1 above)
Comment (JHjava) [reply Gemfile.lock]: Is it an issue that this has been pulled in or is this OK?

Comment (rhowe-gds) [reply Gemfile.lock]: If it builds OK on travis, it'll probably build OK on CI

Comment (JHjava) [reply Gemfile.lock]: 👍

**Pull request merged**